{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -36,6 +36,7 @@
             # Mimicing memcache behavior.
             # http://jazzband.github.io/django-redis/latest/#_memcached_exceptions_behavior
             "IGNORE_EXCEPTIONS": True,
+            "PARSER_CLASS": "redis.connection.HiredisParser",
```

Comment:
If the user choose `windows = 'y'` [this would raise an error](https://github.com/andymccurdy/redis-py/blob/09da3ad07f26350d093929fe1ea6eaaca79ab944/redis/connection.py#L370). 

Given that the point raised by @jcass77, it's probably simpler to leave this out.